node/core/dispute-coordinator/src/participation/mod.rs

```diff
@@ -212,6 +212,29 @@ impl Participation {
 		Ok(())
 	}
 
+	/// Reprioritizes participation requests for disputes that are freshly included
+	pub async fn prioritize_newly_included<Context>(&mut self, ctx: &mut Context, events: &Vec<CandidateEvent>) {
```

Comment:
Turns out this is possible since clearing spam slots for backed candidates already happens in `process_on_chain_votes`. So we only need the included events in the first place.